src/renderer.ts

```diff
@@ -315,7 +315,7 @@ const richLinkStyles = css`
     background: ${neutral[97]};
     padding: ${basePx(1)};
 
-    h1 {
+    h4 {
```

Comment:
Why an `h4`? Are there `h1`, `h2` and `h3`s that sit above this in the hierarchy?

Reply:
Ah yeah should be a h1, was using it for the styles rather than the semantics